examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Maybe we can use this link instead of the RG one: https://www.scielo.br/scielo.php?pid=S1678-58782009000300004&amp;script=sci_arttext</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>